README.md

```diff
@@ -41,31 +35,35 @@ For usage within a web page, paste the following snippet into the header of your
 
 ```html
 <script type="text/javascript">
-!function(e,r,t,s,a){e.__visearch_obj=a;var c=e[a]=e[a]||{};c.q=c.q||[],c.factory=function(r){return function(){var e=Array.prototype.slice.call(arguments);return e.unshift(r),c.q.push(e),c}},c.methods=["idsearch","uploadsearch","colorsearch","set","send","search","recommendation","out_of_stock","similarproducts","discoversearch"];for(var o=0;o<c.methods.length;o++){var n=c.methods[o];c[n]=c.factory(n)}var i=r.createElement(t);i.type="text/javascript",i.async=!0,i.src="//cdn.visenze.com/visearch/dist/js/visearch-1.5.2.min.js";var h=r.getElementsByTagName(t)[0];h.parentNode.insertBefore(i,h)}(window,document,"script",0,"visearch");
-visearch.set('app_key', 'YOUR_APP_KEY');
+!function(t,r,e,a){t.__productcat_obj=a;var c=t[a]=t[a]||{};c.q=c.q||[],c.factory=function(r){return function(){var t=Array.prototype.slice.call(arguments);return t.unshift(r),c.q.push(t),c}},c.methods=["set","similarproducts","productsummary","productdetails"];for(var o=0;o<c.methods.length;o++){var s=c.methods[o];c[s]=c.factory(s)}var n=r.createElement(e);n.type="text/javascript",n.async=!0,n.src="../dist/js/productcat.js";var u=r.getElementsByTagName(e)[0];u.parentNode.insertBefore(n,u)}(window,document,"script","productcat");
+
+productcat.set('app_key', 'YOUR_APP_KEY');
+productcat.set('cid', "YOUR_CID")
 </script>
 ```
 
-This snippet will load `visearch.js` onto the page asynchronously, so it will not affect your page load speed.
+This snippet will load `productcat.js` onto the page asynchronously, so it will not affect your page load speed.
 
-Replace `YOUR_APP_KEY` with your ViSearch Client-side App Key. It is recommended to initiate the client when the SDK is loaded into the page.
+please contact Visenze to get YOUR_APP_KEY and YOUR_CID to put in the field.
 
-Your credentials can be found in [ViSearch Dashboard](https://dashboard.visenze.com)
 
 For usage with Node.js projects:
 
 ```sh
-npm install visearch-javascript-sdk
+npm install productcat-javascript-sdk
 ```
 
 ```js
 // Import module
-import visearch from 'visearch-javascript-sdk';
+import productcat from 'productcat-javascript-sdk';
 
 // Set up keys
-visearch.set('app_key', 'YOUR_APP_KEY');
-visearch.set('timeout', TIMEOUT_INTERVAL_IN_MS); // optional; default value is 15000
-visearch.set('endpoint', YOUR_ENDPOINT); // optional; default value is http://visearch.visenze.com
+productcat.set('app_key', '488c9b80cd314d7ea4af0200e25daf89');
+productcat.set('cid', 1136)
+productcat.set('country', "SG"); // highly recommend, put the country code in the setting can tell productcat only retrun the product available in this country
+productcat.set('timeout', 2000); // optional default is 15000
+
```

Comment:
- can endpoint be overriden? 
- are there support to send custom parameters ? e.g. new API parameters in future that is not available now. This is for old clients to be able to send the request. Also for sending hidden API params.

Reply:
hiden parameters can send directly in the api method. 

prototypes.productsummary = function(params, options, callback, failure) 

params can put any parameters. it will transform into formdata and send out

Reply:
ok that's good